test/plugins/BadPluginAlign.cc

```diff
@@ -15,20 +15,20 @@
  *
 */
 
-#include "ignition/common/PluginMacros.hh"
+#include "gz/common/PluginMacros.hh"
```

Comment:
Ahh I'm undoing the plugin header migration on 

* https://github.com/gazebosim/gz-common/pull/350

We can either merge that first and update this PR, or the other way around. I think it may be easier to do the former so we have less changes to update here. I need to address reviewer feedback on that PR.

Reply:
Let's do the former! I'll be updating this one in the meantime.

Reply:
Rebased after the merge of the PR!

The rebase was not nice to do, and I had to do some manual additions and changes, might want to take another look at the changes:
- https://github.com/gazebosim/gz-common/pull/356/commits/d6211d1739f0736818a08777ddde6cd26fabb94b
- https://github.com/gazebosim/gz-common/pull/356/commits/c6c7825a2c5e7c0d6bb3c721e368108453c02afe